src/components/Lesson/Lesson.js

```diff
@@ -15,6 +18,16 @@ const Lesson = React.createClass({
       __html: this.props.lesson.content
     };
   },
+  componentWillMount(){
```

Comment:
I am not sure if I follow now. Are you confused because `process` is an object with a lot of stuff? I though process was something nodejs used?

Anyway, why not just check if `document` is defined to see if we are running a server process?
```
if (typeof document !== 'undefined') {
  // Client render code goes here...
}
```

Or, in `webpack.static.config.babel.js` just use `webpack.DefinePlugin()` to define a variable to say that we are running the server version?
https://webpack.github.io/docs/list-of-plugins.html#defineplugin